### PR TITLE
Maiorov/create page number 404

### PR DESCRIPTION
**Pull request by maiorovvv**
Please check my rafce for page 404

### CODE REVIEW

File: src/features/NotFound/NotFound.tsx [hunk 1]
```diff
@@ -0,0 +1,30 @@
...
+	return (
+		<section className="error__section section--padding">
+			<div className="container">
+				<div className="row row-cols-1">
+					<div className="col">
+						<div className="error__content text-center">
+							<img
+								className="error__content--img mb-50"
+								src="./src/img/other/404-thumb.png"
+								alt="error-img"
```
Comment (Igor-Klein-1988) [inline src/features/NotFound/NotFound.tsx]: 404 error / page not found

File: src/features/NotFound/NotFound.tsx [hunk 2]
```diff
@@ -0,0 +1,30 @@
...
+const NotFound: React.FC = (): JSX.Element => {
+	return (
+		<section className="error__section section--padding">
+			<div className="container">
+				<div className="row row-cols-1">
+					<div className="col">
+						<div className="error__content text-center">
+							<img
+								className="error__content--img mb-50"
+								src="./src/img/other/404-thumb.png"
```
Comment (Igor-Klein-1988) [inline src/features/NotFound/NotFound.tsx]: use `import image-404 from ''./src/img/other/404-thumb.png"`

Comment (Igor-Klein-1988) [reply src/features/NotFound/NotFound.tsx]: Do you have other 404 pages? (404-somethings)

File: src/features/NotFound/NotFound.tsx (hunk 1 above)
Comment (maiorovvv) [reply src/features/NotFound/NotFound.tsx]: fixit

File: src/features/NotFound/NotFound.tsx (hunk 2 above)
Comment (maiorovvv) [reply src/features/NotFound/NotFound.tsx]: fixit

**Review by DenysBabenko29 — APPROVED** [review]

**Pull request merged**